### ISSUE TITLE
Minor fixes for D-pad and ROM header name

### ISSUE THREAD

**Issue by vaguerant** [emukidid/Wii64#97]
Fix for D-pad Right overlapping with C-buttons:

https://github.com/emukidid/Wii64/blob/546144d5f7a392348cf0aeb13e932db1a09649f0/gc_input/controller-Classic.c#L108
```c
	unsigned int b = (unsigned short)controller->btns;
```
Per Extrems in extremscorner/not64@426ac4d

---
Fix for reading beyond the ROM header title (0x14/20 bytes, not 0x20/32):

https://github.com/emukidid/Wii64/blob/546144d5f7a392348cf0aeb13e932db1a09649f0/main/rom_gc.c#L157-L158
```c
  strncpy(buffer, (char*)ROM_HEADER.Name,20);
  //Maximum ROM name is 20 bytes. Lets make sure we cut off trailing spaces
```
Per https://n64brew.dev/wiki/ROM_Header

**Comment by emukidid** [comment]
Fixed in 192b7c022c3d85f4425ee35b837a562558d1308a, thanks :)

**Comment by saulfabregwiivc** [comment]
This looks promising, thanks!
